target/linux/realtek/files-5.10/drivers/clk/realtek/Kconfig

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: GPL-2.0-only
+config COMMON_CLK_RTL83XX
+        bool "Clock driver for Realtek RTL83XX"
+	depends on MIPS
```

Comment:
This should depend on the RTL83XX architecture here. The driver doesn't apply to MIPS in general.

Reply:
fixed in upcoming version